decimalSuite_test.go

```diff
@@ -50,23 +52,34 @@ func TestFromSuite(t *testing.T) {
 			if PrintFiles {
 				fmt.Println("starting test:", file)
 			}
-			dat, _ := ioutil.ReadFile(file)
-			dataString := string(dat)
-			testVals := getInput(dataString)
-			numTests := len(testVals)
+			f, _ := os.Open(file)
```

Comment:
Is it easy to split context and testing changes into distinct PRs?

Reply:
Split into two branches, test suite PR will be opened when this one is closed